include/grpc++/impl/codegen/status.h

```diff
@@ -48,10 +48,13 @@ class Status {
   Status() : code_(StatusCode::OK) {}
 
   /// Construct an instance with associated \a code and \a error_message
```

Comment:
Considered that, but that will likely pull in all the gpr_log stuff into codegen/ and I feel like it may not worth it...